### ISSUE TITLE
[/templates/schedule] Hour counts are condensed

### ISSUE THREAD

**Issue by aldahick** [crossroads-education/eta-office#6]
Kevin (and Miriah) reported that [this](http://ss.tiin57.net/4Ggf3) is present at 100% zoom, but is fine at 90%.

**Comment by aldahick** [comment]
Fixed by 018721e872ae083c0ab63c6e778c72a89dd9735d

**Comment by aldahick** [comment]
This has resurfaced per the report below:

![Screenshot of message](http://ss.tiin57.net/NScx5)

(This may be another issue as well - waiting on clarification from the user.)

**Comment by aldahick** [comment]
This is not the same issue - on hold until we determine whether or not we're going to handle the request.
